app/views/layouts/manage_cmr.html.erb

```diff
@@ -3,6 +3,7 @@
 <!--[if IE 7]><html class="no-js lt-ie9 lt-ie8"  lang="en"> <![endif]-->
 <!--[if IE 8]><html class="no-js lt-ie9"  lang="en"> <![endif]-->
 <!--[if gt IE 8]><html class="no-js" lang="en"> <!--<![endif]-->
+<html lang="en">
```

Comment:
I see the result of the code from https://mmt.earthdata.nasa.gov/manage_cmr, so yea, this line is needed

    <!DOCTYPE html>
    <!--[if lt IE 7]><html class="no-js lt-ie9 lt-ie8 lt-ie7"  lang="en"> <![endif]-->
    <!--[if IE 7]><html class="no-js lt-ie9 lt-ie8"  lang="en"> <![endif]-->
    <!--[if IE 8]><html class="no-js lt-ie9"  lang="en"> <![endif]-->
    <!--[if gt IE 8]><html class="no-js" lang="en"> <!--<![endif]-->
    <head>
      <title>Metadata Management Tool</title>

      <meta charset="utf-8">

Reply:
Does the `lang="en"` attribute not work in the `<!DOCTYPE html>` tag?

Reply:
I have never seen HTML 5 doc types with language, and w3c says the html team is where we do this at. https://www.w3.org/International/questions/qa-html-language-declarations